src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -71,22 +67,11 @@
                         {{ 'artemisApp.assessmentDashboard.submissions' | artemisTranslate }}
                     </div>
                     <div class="col-5">
-                        <div *ngIf="exercise.isAtLeastInstructor">
-                            <a [routerLink]="submissionsLink">
-                                <span class="text-success">{{ numberOfSubmissions.inTime }}</span>
-                                <span *ngIf="numberOfSubmissions.late > 0"> - </span>
-                                <span *ngIf="numberOfSubmissions.late > 0" class="text-danger">{{ numberOfSubmissions.late }}</span>
-                            </a>
-                            |
-                            <a [routerLink]="submissionsLink" [queryParams]="{ tutorId: tutor?.id }">{{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)</a>
-                        </div>
-                        <div *ngIf="!exercise.isAtLeastInstructor">
-                            <span class="text-success">{{ numberOfSubmissions.inTime }}</span>
-                            <span *ngIf="numberOfSubmissions.late > 0"> - </span>
-                            <span *ngIf="numberOfSubmissions.late > 0" class="text-danger">{{ numberOfSubmissions.late }}</span>
-                            |
-                            <a [routerLink]="submissionsLink" [queryParams]="{ tutorId: tutor?.id }">{{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)</a>
-                        </div>
+                        <span class="text-success">{{ numberOfSubmissions.inTime }}</span>
```

Comment:
probably should be a `<p>` for this entire text

Reply:
I did not change the styling here and would like to keep it that way